public/pages/CreateMonitor/containers/CreateMonitor/utils/constants.js

```diff
@@ -23,6 +22,17 @@ export const FORMIK_INITIAL_WHERE_EXPRESSION_VALUES = {
   fieldRangeEnd: undefined,
 };
 
+export const DEFAULT_ASSOCIATED_MONITORS_VALUE = {
+  sequence: {
+    delegates: [
+      /*{
+        order: 1,
+        monitor_id: '{{m1}}',
```

Comment:
Is this intentional or dev code?

Reply:
That was added as a sample delegate, moving it to JS Doc.